Sources/Common/DataModel/Plane/index.js

```diff
@@ -42,7 +60,13 @@ export function extend(publicAPI, model, initialValues = {}) {
 
   // Object methods
   macro.obj(publicAPI, model);
-  macro.setGet(publicAPI, model, ['bounds']);
+  macro.setGetArray(publicAPI, model, [
+    'origin',
+  ], 3);
+  macro.setGetArray(publicAPI, model, [
+    'normal',
+  ], 3);
```

Comment:
You could have gathered `normal` and `origin` inside that list just to have a single `macro.setGetArray` call.